crux-mir/README.md

```diff
@@ -56,12 +56,47 @@ versions corresponding to the last two `crux-mir` releases.
 [mir-json-readme]: https://github.com/GaloisInc/mir-json#readme
 
 
-## Setup: Build and Install
+## Setup
+
+### Build and Install
 
 Use GHC 9.6, 9.8, or 9.10. From the `crux-mir` directory, run:
 
     $ cabal install exe:crux-mir --overwrite-policy=always
 
+### Binary releases
+
+Both `crux-mir` and `mir-json` comes with nightly builds and stable binary releases. If you want to install the nightly build, do the following:
+
+* Go to the [crux-mir action](https://github.com/GaloisInc/crucible/actions/workflows/crux-mir-build.yml) page
+* Select the last build from `master` and scroll down the page, until you find and download the desired binary release (`crux-mir-macos...` or `crux-mir-ubuntu...`)
+* Unzip the downloaded archive and add it to your path
+* Do the same for `mir-json` by going to the [mir-json action](https://github.com/GaloisInc/mir-json/actions) page
```

Comment:
It might make more sense to put this part in the `Setup: `mir-json` section instead.

Reply:
Updated